redbot/cogs/audio/core/events/dpy.py

```diff
@@ -12,19 +14,145 @@
+    "add_reactions": _("Add Reactions"),
+    "view_audit_log": _("View Audit Log"),
+    "priority_speaker": _("Priority Speaker"),
+    "stream": _("Go Live"),
+    "read_messages": _("Read Text Channels & See Voice Channels"),
+    "send_messages": _("Send Messages"),
+    "send_tts_messages": _("Send TTS Messages"),
+    "manage_messages": _("Manage Messages"),
+    "embed_links": _("Embed Links"),
+    "attach_files": _("Attach Files"),
+    "read_message_history": _("Read Message History"),
+    "mention_everyone": _("Mention @everyone, @here, and All Roles"),
+    "external_emojis": _("Use External Emojis"),
+    "view_guild_insights": _("View Server Insights"),
+    "connect": _("Connect"),
+    "speak": _("Speak"),
+    "mute_members": _("Mute Members"),
+    "deafen_members": _("Deafen Members"),
+    "move_members": _("Move Members"),
+    "use_voice_activation": _("Use Voice Activity"),
+    "change_nickname": _("Change Nickname"),
+    "manage_nicknames": _("Manage Nicknames"),
+    "manage_roles": _("Manage Roles"),
+    "manage_webhooks": _("Manage Webhooks"),
+    "manage_emojis": _("Manage Emojis"),
+}
+
+DANGEROUS_COMMANDS = {
+    "command_llsetup_java": _(
+        "This command will change the executable path of Java, "
+        "this is useful if you have multiple installations of Java and the default one is causing issues. "
+        "Please don't change this unless you are certain that the Java version you are specifying is supported by Red. "
+        "The default and supported version is currently Java 11."
+    ),
+    "command_llsetup_heapsize": _(
+        "This command will change the maximum RAM allocation for the managed Lavalink node, "
+        "usually you will never have to change this, "
+        "before considering changing it please consult our support team."
+    ),
+    "command_llsetup_external": _(
+        "This command will disable the managed Lavalink node, "
+        "if you toggle this command you must specify an external Lavalink node to connect to, "
+        "if you do not do so Audio will stop working."
+    ),
+    "command_llsetup_host": _(
+        "This command is used to specify the IP which will be used by Red to connect to an external Lavalink node. "
+    ),
+    "command_llsetup_password": _(
+        "This command is used to specify the authentication password used by Red to connect to an "
+        "external Lavalink node."
+    ),
+    "command_llsetup_secured": _(
+        "This command is used toggle between secured and unsecured connections to an external Lavalink node."
+    ),
+    "command_llsetup_wsport": _(
+        "This command is used to specify the connection port used by Red to connect to an external Lavalink node."
+    ),
+    "command_llsetup_config_host": _(
+        "This command specifies which network interface and IP the managed Lavalink node will bind to, "
+        "by default this is 'localhost', "  # meaning it will bind to all interfaces
```

Comment:
Isn't the comment contradicting what the string is saying? Binding to `localhost` means it's only binded to loopback interface (127.0.0.1), not all of them, right?